.github/workflows/notify-on-review-wanted.yml

```
@@ -0,0 +1,40 @@
+name: Notify on Review Wanted
+on:
+  issues:
+    types: [labeled]
+  pull_request_target:
+    types: [labeled]
+
+permissions:
+  contents: read
```

```suggestion
permissions: {}
```

We aren't checking anything out, so this doesn't need any permissions, right?

---

I'm just following the pattern of notify-on-push.

---

SGTM